pkgs/by-name/we/weevely/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, python3
+, python3Packages
+, makeWrapper
+, installShellFiles
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "weevely";
+  version = "4.0.1-unstable-2023-06-18";
+  format = "other";
```

Comment:
As far as I know, format is deprecated (can't prove it right now), consider using `pyproject = false;`, see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

Reply:
Done